src/docs/asciidoc/developer-manual/develop-cloud-services/initialization-code.adoc

```diff
@@ -0,0 +1,27 @@
+:relative-path: ../../
+include::{docdir}/variables.adoc[]
+
+ZetaPush provides you the possibility to add some initialization code in your _custom cloud service_.
+For this you need to implement the method `onApplicationBootstrap()` in your _custom cloud service_.
+
+.Initialization code
+[source, javascript, linenums]
+----
+import { Bootstrappable } from '@zetapush/core';
+
+export default class implements Bootstrappable {
+  async onApplicationBootstrap() {
+    console.log('Initialization');
```

Comment:
We should have a more complex sample with configuration of a service (+ injection in constructor) and also save some data for example